lib/diagnostic.rb

```diff
@@ -31,31 +33,32 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = '#{:character} is really a Sith Lord.'
 
```

Comment:
The only thing here is we don't need the colon.
This would actually print "character is really a Sith Lord."